.github/workflows/test-runner-nightly-perf.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Sample Nightly Perf Tests
```

Comment:
Nit: Not sure if the Sample was a leftover or intended?

Reply:
Replaced this with comments suggested.